### DISCUSSION TITLE
How to read the minimum value of S11 using PyAEDT

### DISCUSSION THREAD

**Discussion by ghost** [ansys/pyaedt#3800] [Q&A] [answered]
Hi,

Currently, I'm researching how to controll HFSS using PyAEDT.
In an antenna design, I would like to know how to read the minimum value of S11 using PyAEDT.
Does anybody know the code?
I don't know it, so please tell me how to look up the code.

Thank you.

**Comment by Samuelopez-ansys** [comment, marked as answer]
Hi @crapemyrtle-841,

There are different ways, but I would use the same approach as this example:

https://aedt.docs.pyansys.com/version/stable/examples/02-HFSS/Probe_Fed_Patch.html#sphx-glr-examples-02-hfss-probe-fed-patch-py

As you can see, the S11 can easily obtained:
https://aedt.docs.pyansys.com/version/stable/examples/02-HFSS/Probe_Fed_Patch.html#plot-s11

Then you can manipulate the data using python.

**Reply by ghost** [reply]
Hi @Samuelopez-ansys 

Thank you for your response.
I would like to get only the minimum value, but you mean I need to read all the S11 data once?

**Reply by Samuelopez-ansys** [reply]
@crapemyrtle-841 Yes you need to get S11 data, and then with python very easily you can get any information of the data, I encourage you debug the previous example, and you will how you can get S11 data.

You can also pass the expression min(dB(S1,1)) and you will only get the minimum directly, it is up to you. Please play a bit with PYAEDT and you will get easily how it works.

**Reply by ghost** [reply]
@Samuelopez-ansys 
I understand.
Thank you for your kind explanation!
